src/Entities/Entity.cpp

```diff
@@ -1632,8 +1632,11 @@ void cEntity::TeleportToEntity(cEntity & a_Entity)
 
 void cEntity::TeleportToCoords(double a_PosX, double a_PosY, double a_PosZ)
 {
-	SetPosition(a_PosX, a_PosY, a_PosZ);
-	m_World->BroadcastTeleportEntity(*this);
+	if (!cRoot::Get()->GetPluginManager()->CallHookEntityTeleport(*this, m_LastPos, GetPosition()))  // if the Plugin doesn't denie the teleport
```

Comment:
`deny` is the proper spelling. Also, it might be a good idea to put the comment above the condition, so that the line is not that long. I'd write it like this:
```cpp
// If plugins agree with the teleport, set the new position:
if (!cRoot::Get()->...)
```